pkg/virt-handler/options.go

```diff
@@ -119,3 +122,16 @@ func disksInfoToDisksInfo(disksInfo map[string]*containerdisk.DiskInfo) map[stri
 	}
 	return info
 }
+
+func convertSiblings(siblings string) []uint32 {
```

Comment:
Is this logic covered by tests?

Reply:
So `options.go` in its entirety doesn't have tests AFAIK.

What could be done is add capabilities to `vm_test.go` controller creation (it's currently passing nil for capabilities) and then it would be covered by the call to `Execute` during tests. 

I'm a bit cautious about adding this though, let me know what you think.

Reply:
I think we could add `options_test.go` and test at least `capabilitiesToTopology` in order to verify that the change introduced by this PR hasn't caused a regression.